pkg/cmd/serve/cmd.go

```diff
@@ -20,6 +22,8 @@ func NewServeCommand(env *environments.Env) *cobra.Command {
 			if err != nil {
 				glog.Fatalf("Unable to initialize environment: %s", err.Error())
 			}
+			info := information.GetBuildInfo()
+			glog.Infof("GoVersion: %s. Commit time: %s. Architecture: %s. Operating System: %s. VCS Type: %s. CommitSha: %s.", info.GetGoVersion(), info.GetVCSTime(), info.GetArchitecture(), info.GetOperatingSystem(), info.GetVCSType(), info.GetCommitSha())
```

Comment:
I suggest changing `%s` by `%q`. `%q` adds single quotes around the value. Due to some of the returned values can be the empty string having single quotes around them is useful.